browser-test/src/admin/admin_application_statuses.test.ts

```diff
@@ -225,11 +225,11 @@ test.describe('view program statuses', () => {
           const modal =
             await adminPrograms.setStatusOptionAndAwaitModal(emailStatusName)
           const notifyCheckbox = await modal.$('input[type=checkbox]')
-          if (!notifyCheckbox) {
-            throw new Error('Expected a checkbox input')
+          expect(notifyCheckbox).not.toBeNull()
+          if (notifyCheckbox) {
+            await notifyCheckbox.uncheck()
+            expect(await notifyCheckbox.isChecked()).toBe(false)
           }
```

Comment:
```suggestion
          expect(notifyCheckbox).not.toBeNull()
          await notifyCheckbox.uncheck()
          expect(await notifyCheckbox.isChecked()).toBe(false)
```

Since this is now asserting that `notifyCheckbox` is not null you don't need the `if` block.

Reply:
you would think that, but when i try to remove it i get an "is possible null" error 
![image](https://github.com/civiform/civiform/assets/24319951/5f304b30-eb82-4205-955c-04679f55031a)

Reply:
Use `notifyCheckbox!.`

Reply:
Or keep the `if` block for now

Reply:
Ohh good idea! TIL about typescript's `!`. Updated.